ui/app/AppLayouts/Profile/views/PrivacyView.qml

```diff
@@ -166,10 +166,15 @@ Item {
                 StatusQControls.StatusSwitch {
                     id: switch1
                     checked: !localAccountSensitiveSettings.onlyShowContactsProfilePics
+                    onCheckedChanged: {
+                        if (localAccountSensitiveSettings.onlyShowContactsProfilePics !== !checked) {
```

Comment:
Not a big deal, but the double negative is a bit ugly. Something like
```qml
if (localAccountSensitiveSettings.onlyShowContactsProfilePics === checked) {
```
is prettier and easier to read

Reply:
@jrainville you know its not true :) this settings is ugly bc flag is inverted. All conditions now is
`if (settings.value != toggle.state) settings.value = toggle.state`. But for several, `toggle.state` is inverted.
i see two more `pretty` solutions:
1. rename `onlyShowContactsProfilePics` to `displayAllProfilePics` for `view` level (minimum).
2. Add `inverted` state to `StatusSwitch`. Which means that visually if `checked == true` toggle looks like `checked == false`.
What do you think?

Reply:
I agree that the naming is a bit confusing since it's "negative" in a way. We can rename it as you said if you want.
But also, I think just removing the two `!` works, because 
```js
localAccountSensitiveSettings.onlyShowContactsProfilePics !== !checked
// true !== false  => true
```
is the same as
```
localAccountSensitiveSettings.onlyShowContactsProfilePics === checked
// true === true => true
```
and the latter is way easier to read